docker/Dockerfile-dhvirtualenv

```diff
@@ -55,17 +52,18 @@ ENV distro ${distro}
 # http://bugs.python.org/issue19846
 ENV LANG C.UTF-8
 
+COPY --from=builder /dh-virtualenv_*.deb /
+
```

Comment:
this was after most of the apt-gets to help with faster iteration, but :man_shrugging:

Reply:
Sure, but moving it up here lets us remove an entire call to `apt-get update && apt-get install`, so I'd expect this to still be faster than the previous setup. And this is just dh-virtualenv itself, which shouldn't need to change much (ever?) between builds.

Reply:
to be clear: it was here to help iteration when working on the first half of the Dockerfile (where it builds dh-virtualenv).

Yes it will be faster in the full-rebuild case, but not when iterating on that.

Anyway, it's all fine. Just thought I'd share the background.